app/1.0/blog/2016-10-03-1.7-release.md

```diff
@@ -0,0 +1,25 @@
+---
+title:  "1.7 Release"
+
+---
+_We announce significant releases here on the blog, highlighting the major new features as well as breaking changes. As always, we'd love to get your feedback, issues, and PR's, so tweet us [@polymer](https://twitter.com/polymer) or contribute to the [Github](https://github.com/Polymer)._
+
+We're pleased to announce the release of:
+
+## Polymer 1.7
+
+The primary goal of the upcoming Polymer 2.0 release is to take advantage of native, cross-browser support for Web Components. With this in mind, Polymer 1.7.0 provides forwards compatibility and initial support for migration. 
+
+### `<slot>` element 
+
+Elements can opt-in to using the new `<slot>` element and `::slotted()` CSS selector from ShadowDOM v1.
+
+This is not required for current Polymer 1.x elements, but using the new syntax will make it easier to port elements to 2.x.
```

Comment:
2.0, which requires the new syntax. 

I think maybe we should move this up to the general section because it really describes both of these changes.

(That is, both of these features _allow you to use 2.0 compatible syntax in 1.x elements_.)